app/models/user.rb

```diff
@@ -11,8 +11,16 @@ class User < ApplicationRecord
     customer: CUSTOMER = 'customer'
   }
 
+  mount_uploader :avatar, AvatarUploader
+
   validates :first_name, presence: true, length: { maximum: 50 }
   validates :last_name, presence: true, length: { maximum: 50 }
   validates :email, format: { with: URI::MailTo::EMAIL_REGEXP },
                     uniqueness: true, length: { maximum: 255 }
+
+  paginates_per 9
+
+  def full_name
```

Comment:
is not a part of business logic, move to decorator as you need to have them anyway

Reply:
done